src/Runner.Worker/Handlers/HandlerFactory.cs

```diff
@@ -68,6 +68,28 @@ public sealed class HandlerFactory : RunnerService, IHandlerFactory
                     bool isOptOut = isWorkflowOptOutSet ? StringUtil.ConvertToBoolean(workflowOptOut) : isLocalOptOut;
                     if (!isOptOut)
                     {
+                        var repoAction = action as Pipelines.RepositoryPathReference;
+                        if (repoAction != null)
+                        {
+                            var warningActions = new HashSet<string>();
+                            if (executionContext.Global.Variables.TryGetValue("Node16ForceActionsWarnings", out var node16ForceWarnings))
```

Comment:
Could we extract string `Node16ForceActionsWarnings` to constants? :) https://github.com/actions/runner/blob/d4bbbb841987d6c2d4830b05e124735ac1f6d6be/src/Runner.Common/Constants.cs#L157